share/admin.py

```diff
@@ -227,14 +241,22 @@ def end_date_(self, obj):
         return obj.end_date.date().isoformat()
 
     def status_(self, obj):
-        if obj.status == HarvestLog.STATUS.created and (timezone.now() - obj.date_modified) > datetime.timedelta(minutes=60):
-            return '<span style="font-weight: bold;">Lost</span>'
+        if obj.status == HarvestLog.STATUS.created and (timezone.now() - obj.date_modified) > datetime.timedelta(days=1, hours=6):
+            return format_html('<span style="font-weight: bold;">Lost</span>')
```

Comment:
`lost` is something that hasn't had its status changed in a 1.25 days. IE the task was created but was never run. I don't think it qualifies as a status, there is nothing that triggers the lost state. It's just an assumption/something to quickly draw the eye.